index.html

```diff
@@ -0,0 +1,362 @@
+                <img src="images/facebook_f_64px.png" />
+              </figure>
+            </li>
+            <li>
+              <figure class="image is-24x24">
+                <img src="images/twitter_64px.png" />
+              </figure>
+            </li>
+            <li id="eng">English</li>
+            <li id="mypage">MyPage</li>
+            <li id="logout">Logout</li>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <!-- navbar section -->
+
+    <nav
+      class="navbar pl-4 pr-4 is-flex"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div class="navbar-brand is-flex">
+        <a class="navbar-item" href="https://google.com">
+          <div class="logo is-hidden-mobile">
+            <span class="black">CCG-</span>
+            <span class="red">CON</span>
+          </div>
+        </a>
+
+        <a
+          role="button"
+          class="navbar-burger"
+          aria-label="menu"
+          aria-expanded="false"
+          data-target="navbarBasicExample"
+        >
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+        </a>
+      </div>
+
+      <div id="navbarBasicExample" class="navbar-menu ">
+        <div class="navbar-start"></div>
+
+        <div class="navbar-end">
+          <a class="navbar-item" href="about.html"> About </a>
+
+          <a class="navbar-item"> Program </a>
+
+          <a class="navbar-item"> Join </a>
+          <a class="navbar-item"> Sponsor </a>
+          <a class="navbar-item"> News </a>
+
+          <div class="navbar-item">
+            <div class="buttons">
+              <a class="button is-outlined red"> Campaign </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+
+    <div class="main">
+      <div class="container pt-3">
+        <h2 class="title is-2 mt-0">Hello Crypto Family</h2>
+         <h1>
+          <div class="transparent is-flex-shrink-5 mt-0">
+            CRYPTO CURRENCY <br />
+            GLOBAL CONFERENCE<br />
+            2023
+          </div>
+        </h1>
+        <section class="section pl-0 pr-0">
+          <div class="columns pt-3">
+            <div class="column">
+              <div class="box">
+                Welcome to the Global Cryptocurrency Conference 2023, the premier gathering of innovators, experts, and enthusiasts, charting the future of digital currencies and blockchain technology.
+              </div>
+            </div>
+          </div>
+
+          <h3 class="subtitle is-3 mt-2">2023.10.15(THU) ~ 16(FRI) <br /></h3>
+          <p id="location">Las Vegas, Nevada</p>
+        </section>
+      </div>
+      <div class="program pb-6">
+        <div class="columns is-centered">
+          <div class="column is-narrow has-text-centered">
+            <h4
+              class="title is-4 has-text-white mt-2 mb-1 has-text-centered"
+              id="mainpg"
+            >
```

Comment:
- [ ] Several issues need attention in the headline section:
   - The font size used for the title of this section appears overly large, causing certain parts of it to be obscured towards the right.
   - In line with the original design template, the text - **Welcome to the Global Cryptocurrency Conference 2023, ...** is meant to be enclosed within a container featuring a white border. This container is intentionally designed to avoid spanning the full width of the screen. However, your current design does not adhere to this arrangement. Kindly address these discrepancies accordingly.


Original design
<img width="223" alt="image" src="https://github.com/Drew246/capstone-project-1/assets/84156714/fd64fe1a-1338-45e6-b648-5ae49f0cfe62">

Your project
![image](https://github.com/Drew246/capstone-project-1/assets/84156714/d8416814-7822-4a54-b427-9a167d45601f)